src/Knp/DictionaryBundle/Resources/config/services.yaml

```diff
@@ -0,0 +1,19 @@
+---
+imports:
+  - resource: 'services/debug.yaml'
+  - resource: 'services/factories.yaml'
+  - resource: 'services/faker.yaml'
+  - resource: 'services/forms.yaml'
+  - resource: 'services/templating.yaml'
+  - resource: 'services/transformers.yaml'
+  - resource: 'services/validation.yaml'
+
+services:
+
+  knp_dictionary.dictionary.dictionary_registry:
```

Comment:
Why are we not using class namespaces as service ids  ?

Reply:
Next step dude ;)